app/src/main/java/nyc/pursuit/ttptwitterbuild/MainActivity.java

```diff
@@ -30,21 +30,54 @@
     implements NavigationView.OnNavigationItemSelectedListener {
 
   private static final String TAG = "GPS";
+
+  // Why do you have public static objects?
```

Comment:
You should almost never have public static objects. If you need multiple classes to access an object, pass them in.